Tools/Scripts/webkitpy/benchmark_runner/http_server_driver/http_server/builtin_http_server.py

```diff
@@ -6,6 +6,9 @@
 import sys
 import threading
 import socket
+import json
+
+import signposts # this import will succeed even if signposts module is not compiled
```

Comment:
As noted offline, this import should get moved to `do_POST`, although we should also import it right after we do argument parsing if we're going to use the module, since we don't want to on-the-fly compile the module when our server is responding to a request.